dynawo/sources/Models/Modelica/Dynawo/Examples/IEEE118/BaseClasses/IEEE118Base.mo

```diff
@@ -0,0 +1,1117 @@
+    Dynawo.Electrical.Lines.Line L_110_112_1(BPu = 0.0 * ZBASE138_0, GPu = 0.0 * ZBASE138_0, RPu = 4.703868 / ZBASE138_0, XPu = 12.18816 / ZBASE138_0);
+    Dynawo.Electrical.Lines.Line L_114_115_1(BPu = 0.0 * ZBASE138_0, GPu = 0.0 * ZBASE138_0, RPu = 0.438012 / ZBASE138_0, XPu = 1.9805760000000001 / ZBASE138_0);
+
+     //Transformers
+    Dynawo.Electrical.Transformers.TransformerFixedRatio T_8_5_1(BPu = 0.0 * ZBASE138_0, GPu = 0.0 * ZBASE138_0, RPu = 0.0 / ZBASE138_0, XPu = 5.084748 / ZBASE138_0, rTfoPu = 1.015228426395939);
+    Dynawo.Electrical.Transformers.TransformerFixedRatio T_30_17_1(BPu = 0.0 * ZBASE138_0, GPu = 0.0 * ZBASE138_0, RPu = 0.0 / ZBASE138_0, XPu = 7.3890720000000005 / ZBASE138_0, rTfoPu = 1.0416666666666667);
+    Dynawo.Electrical.Transformers.TransformerFixedRatio T_26_25_1(BPu = 0.0 * ZBASE138_0, GPu = 0.0 * ZBASE138_0, RPu = 0.0 / ZBASE138_0, XPu = 7.274807999999999 / ZBASE138_0, rTfoPu = 1.0416666666666667);
+    Dynawo.Electrical.Transformers.TransformerFixedRatio T_38_37_1(BPu = 0.0 * ZBASE138_0, GPu = 0.0 * ZBASE138_0, RPu = 0.0 / ZBASE138_0, XPu = 7.1415 / ZBASE138_0, rTfoPu = 1.06951871657754);
+    Dynawo.Electrical.Transformers.TransformerFixedRatio T_63_59_1(BPu = 0.0 * ZBASE138_0, GPu = 0.0 * ZBASE138_0, RPu = 0.0 / ZBASE138_0, XPu = 7.350984 / ZBASE138_0, rTfoPu = 1.0416666666666667);
+    Dynawo.Electrical.Transformers.TransformerFixedRatio T_64_61_1(BPu = 0.0 * ZBASE138_0, GPu = 0.0 * ZBASE138_0, RPu = 0.0 / ZBASE138_0, XPu = 5.103792 / ZBASE138_0, rTfoPu = 1.015228426395939);
+    Dynawo.Electrical.Transformers.TransformerFixedRatio T_65_66_1(BPu = 0.0 * ZBASE138_0, GPu = 0.0 * ZBASE138_0, RPu = 0.0 / ZBASE138_0, XPu = 7.046279999999999 / ZBASE138_0, rTfoPu = 1.06951871657754);
+    Dynawo.Electrical.Transformers.TransformerFixedRatio T_68_69_1(BPu = 0.0 * ZBASE138_0, GPu = 0.0 * ZBASE138_0, RPu = 0.0 / ZBASE138_0, XPu = 7.046279999999999 / ZBASE138_0, rTfoPu = 1.06951871657754);
+    Dynawo.Electrical.Transformers.TransformerFixedRatio T_81_80_1(BPu = 0.0 * ZBASE138_0, GPu = 0.0 * ZBASE138_0, RPu = 0.0 / ZBASE138_0, XPu = 7.046279999999999 / ZBASE138_0, rTfoPu = 1.06951871657754);
+
+     //Frequency
+    Dynawo.Electrical.Controls.Frequency.SignalN Model_Signal_N;
+    Dynawo.Types.Angle Theta_B49;
+
+     //Generators
+    Dynawo.Electrical.Machines.SignalN.GeneratorPVTfo B1_G1(KGover = 0.0, PGen0Pu = 0.0, PMaxPu = 100.0, PMinPu = -100.0, PNom = 100.0, PRef0Pu = -0.0, QGen0Pu = -0.031, QMaxPu = 0.15, QMinPu = -0.05, QNomAlt = 15.0, QStator0Pu = -0.2069, SNom = 10000.0, U0Pu = 1.0, UStator0Pu = 1.0, UStatorRef0Pu = 0.955, XTfoPu = 0.1228, SwitchOffSignal10 = false, SwitchOffSignal20 = false, SwitchOffSignal30 = false, limUQDown0 = false, limUQUp0 = false, qStatus0 = Dynawo.Electrical.Machines.SignalN.GeneratorPV.QStatus.Standard, i0Pu = Complex(0.0, -0.031), iStator0Pu = Complex(0.0, 0.0003), sStator0Pu = Complex(0.0, -0.0003), u0Pu = Complex(1.0, 0.0), uStator0Pu = Complex(1.0, 0.0));
```

Comment:
Why did you choose a GeneratorPVTfo model instead of GeneratorPV

Reply:
This example was generated based on the [iidm file of the IEEE118 nrt](https://github.com/dynawo/dynawo/tree/master/nrt/data/IEEE118) where the generators nominal voltages are above the voltage level for which generator transformers are considered to be in the iidm file (100 kV).